.github/PULL_REQUEST_TEMPLATE.md

```
@@ -107,3 +122,10 @@ bun test
 - ✅ Commit messages: Follow conventional format
 - ✅ No secrets: Security scan
 - ✅ PR size: Auto-labeled
+
+---
+
+## 🏷️ Quick Labels
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)